app/models/tag.rb

```diff
@@ -0,0 +1,3 @@
+class Tag < ApplicationRecord
```

Comment:
Does this ever get used?

Reply:
this was being used briefly in the migration oddly enough, I was able to get rid of it when I cleaned up the migration (something I meant to go back for). I am testing the fixes on the migration changes now.